Client/Frontend/Sync/WebFilter/Bookmarks.html

```diff
@@ -0,0 +1,99 @@
+<!--
+Copyright (c) 2020 The Brave Authors. All rights reserved.
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this file,
+You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+
+<head>
+    <meta name='viewport' content='initial-scale=1, viewport-fit=cover'>
```

Comment:
if this supports content security policy, we should add one like `<meta http-equiv="Content-Security-Policy" content="default-src 'self'">` (only allow resources loaded from the same origin as this file, disable eval and inline scripts)

Reply:
Fixed.